test/integration/scheduler/queueing/queue.go

```diff
@@ -76,6 +79,8 @@ type CoreResourceEnqueueTestCase struct {
 	// EnableSchedulingQueueHint indicates which feature gate value(s) the test case should run with.
 	// By default, it's {true, false}
 	EnableSchedulingQueueHint sets.Set[bool]
+	// OverridingSchedulerProfile is the scheduler profile overriding the default scheduler profile.
+	OverridingSchedulerProfile *configv1.KubeSchedulerProfile
```

Comment:
No need to say "Override" because empty config means default config either way.

```suggestion
	// SchedulerProfile is the scheduler profile to be used within the test case.
	// If empty, the default profile will be used.
	SchedulerProfile *configv1.KubeSchedulerProfile
```